### ISSUE TITLE
After 509700 steps, am still getting bad results using TF 0.12 and training on gtx1070

### ISSUE THREAD

**Issue by andrenatal** [llSourcell/tensorflow_chatbot#19]
I am using the default parameters (256 cells and 3 layers net).

> what were the best movies from 2016? 
We are dry another next places . 
> what were the best movies in 2016? 
We were as less , exactly children . 
> who are you? Come on Marcus . 
> hello? _UNK﻿

**Comment by 1228337123** [comment]
Maybe you dataset is too small. Could you tell us how you run successfully?  Explaining more details will be best. ^_^

**Comment by AlvinChen13** [comment]
I have trained about 170000 steps about 24h with default parameters also(256 cells and 3 layers net), and the perplexity is 1.59, learning rate is 0.1983. And the result is bad. My box is installed with 2 K40c, but only one is used, and the GPU usage is < 40%, and sometimes GPU is stopped, but one CPU core is pretty busy (100%).
 
@andrenatal Have you tested some simple chat? And what's the perplexity after 500000+?

Reading model parameters from working_dir/seq2seq.ckpt-175200
> hello
That ' s right .
> hi

> hi

> hello
That ' s right .
> what's your name?
Pris .
> how old are you?
Twenty-eight .
> where are you?
Screw the back see my call .
> how much is it?
Two at 00 .
> how are you?
I ' m m .
> how are doing?
_UNK .
> how are you?
I ' m m .
> how do you do?
Good I ' m a darling , darling .
> hi, Pris.
_UNK .
> Hi

> hi

> Hello
Just wanted to worry .
> Good morning.
Horses .
> what were the best movies of 2016?
Oh . . . yeah . Well . . . .
> who are you?
Right .
> quit
Let me see the check .
> who are you?
Right .
> what's your name?
Pris .

**Comment by andrenatal** [comment]
Sorry but I gave up of this implementation. After train for days with the
Cornell data set never managed to get decent results so I decided to
abandon it.

On Feb 14, 2017 6:03 PM, "AlvinChen13" <notifications@github.com> wrote:

I have trained about 170000 steps about 24h with default parameters
also(256 cells and 3 layers net), and the perplexity is 1.59, learning rate
is 0.1983. And the result is bad. My box is installed with 2 K40c, but only
one is used, and the GPU usage is < 40%, and sometimes GPU is stopped, but
one CPU core is pretty busy (100%).

@andrenatal <https://github.com/andrenatal> Have you tested some simple
chat? And what's the perplexity after 500000+?

Reading model parameters from working_dir/seq2seq.ckpt-175200

hello
That ' s right .
hi

hi

hello
That ' s right .
what's your name?
Pris .
how old are you?
Twenty-eight .
where are you?
Screw the back see my call .
how much is it?
Two at 00 .
how are you?
I ' m m .
how are doing?
_UNK .
how are you?
I ' m m .
how do you do?
Good I ' m a darling , darling .
hi, Pris.
_UNK .
Hi

hi

Hello
Just wanted to worry .
Good morning.
Horses .
what were the best movies of 2016?
Oh . . . yeah . Well . . . .
who are you?
Right .
quit
Let me see the check .
who are you?
Right .
what's your name?
Pris .

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/llSourcell/tensorflow_chatbot/issues/19#issuecomment-279896295>,
or mute the thread

**Comment by Graystripe17** [comment]
I had a thought: Can you guys make a fork and post the stepdata located in the working_dir? We're all using the same Cornell data & idk but I think the algorithm is determinate. I would offer to help post mine, but I'm not very far.

**Comment by Rabisha** [comment]
I am also getting useless output,can anyone tell me what is the problem?
